Cargo.toml~

```diff
@@ -0,0 +1,218 @@
+libp2p-noise = { version = "0.40.0", path = "transports/noise", optional = true }
+libp2p-ping = { version = "0.40.1", path = "protocols/ping", optional = true }
+libp2p-plaintext = { version = "0.37.0", path = "transports/plaintext", optional = true }
+libp2p-pnet = { version = "0.22.1", path = "transports/pnet", optional = true }
+libp2p-relay = { version = "0.13.0", path = "protocols/relay", optional = true }
+libp2p-rendezvous = { version = "0.10.0", path = "protocols/rendezvous", optional = true }
+libp2p-request-response = { version = "0.22.1", path = "protocols/request-response", optional = true }
+libp2p-swarm = { version = "0.40.1", path = "swarm" }
+libp2p-swarm-derive = { version = "0.30.1", path = "swarm-derive" }
+libp2p-uds = { version = "0.36.0", path = "transports/uds", optional = true }
+libp2p-wasm-ext = { version = "0.37.0", path = "transports/wasm-ext", optional = true }
+libp2p-yamux = { version = "0.41.0", path = "muxers/yamux", optional = true }
+multiaddr = { version = "0.14.0" }
+parking_lot = "0.12.0"
+pin-project = "1.0.0"
+smallvec = "1.6.1"
+
+[target.'cfg(not(any(target_os = "emscripten", target_os = "wasi", target_os = "unknown")))'.dependencies]
+libp2p-deflate = { version = "0.37.0", path = "transports/deflate", optional = true }
+libp2p-dns = { version = "0.37.0", path = "transports/dns", optional = true }
+libp2p-mdns = { version = "0.41.0", path = "protocols/mdns", optional = true }
+libp2p-tcp = { version = "0.37.0", path = "transports/tcp", optional = true }
+libp2p-websocket = { version = "0.39.0", path = "transports/websocket", optional = true }
+
+[target.'cfg(not(target_os = "unknown"))'.dependencies]
+libp2p-gossipsub = { version = "0.42.1", path = "protocols/gossipsub", optional = true }
+
+[dev-dependencies]
+async-std = { version = "1.6.2", features = ["attributes"] }
+async-trait = "0.1"
+env_logger = "0.9.0"
+clap = { version = "4.0.13", features = ["derive"] }
+tokio = { version = "1.15", features = ["io-util", "io-std", "macros", "rt", "rt-multi-thread"] }
+
+[workspace]
+members = [
+    "core",
+    "misc/metrics",
+    "misc/multistream-select",
+    "misc/rw-stream-sink",
+    "misc/keygen",
+    "misc/prost-codec",
+    "misc/quickcheck-ext",
+    "muxers/mplex",
+    "muxers/yamux",
+    "protocols/dcutr",
+    "protocols/autonat",
+    "protocols/floodsub",
+    "protocols/gossipsub",
+    "protocols/rendezvous",
+    "protocols/identify",
+    "protocols/kad",
+    "protocols/mdns",
+    "protocols/ping",
+    "protocols/relay",
+    "protocols/request-response",
+    "swarm",
+    "swarm-derive",
+    "transports/deflate",
+    "transports/dns",
+    "transports/noise",
+    "transports/plaintext",
+    "transports/pnet",
+    "transports/tcp",
+    "transports/uds",
+    "transports/websocket",
+    "transports/wasm-ext",
+<<<<<<< HEAD
+=======
+    "transports/webrtc",
+>>>>>>> eeca244c (feat(interop-tests): build test binary inside container (#3441))
```

Comment:
Merge conflict? Also is this file (`Cargo.toml~`) supposed to be checked in?

Reply:
It is not, not sure what went wrong there :thinking: